modules/rosa-sts-creating-cluster.adoc

```diff
@@ -0,0 +1,194 @@
+[id="rosa-sts-creating-cluster_{context}"]
+= Creating your cluster for using STS
```

Comment:
s/ for using STS / using STS

The cluster is not being created for the purpose of STS, but rather the cluster is being created for the purpose of making applications available.